spec/services/exports/export_distributions_csv_service_spec.rb

```diff
@@ -95,6 +96,7 @@
           distribution.line_items.where(item_id: item_id).total,
           distribution.cents_to_dollar(distribution.line_items.total_value),
           distribution.delivery_method,
+          "$0.00",
```

Comment:
Can we validate that an actual number shows up here?

Reply:
done